templates/data/valkey.html

```diff
@@ -0,0 +1,440 @@
+        <div class="p-equal-height-row__item">
+          <p>
+            <a href="/data/kafka" aria-label="Learn more about Kafka">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/82e94ddc-postgresql.png",
+                        alt="",
+                        width="568",
+                        height="321",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+            }}
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <p>
+            <a href="/data/postgresql" aria-label="Learn more about PostgreSQL">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/23ae1c13-mongodb.png",
+                        alt="",
+                        width="568",
+                        height="321",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+            }}
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <p>
+            <a href="/data/mongodb" aria-label="Learn more about mongoDB">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/b5ec835d-mysql.png",
+                        alt="",
+                        width="568",
+                        height="321",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+            }}
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <p>
+            <a href="/data/mysql" aria-label="Learn more about MySQL">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Learn more about Valkey&trade;</h2>
+      </div>
+      <div class="col">
+        <ul class="p-list--divided">
+          <li class="p-list__item u-no-padding">
+            <p>
+              <a href="https://ubuntu.com/blog/join-canonical-in-vienna-for-valkey-developer-day-2024">Meet Canonical at Valkey Developer Day 2024 in Vienna</a>
```

Comment:
the links in copy doc are different for this section

Reply:
Thanks, I didn't notice the copy was changed this morning. Updated all except one link because it seems like a strange addition, left a comment for pdm on the doc